api/orchestration/v1alpha1/kvcache_types.go

```diff
@@ -104,6 +104,26 @@ type CacheSpec struct {
 	CPU string `json:"cpu,omitempty"`
 }
 
+type AffinityConfig struct {
```

Comment:
we can use annotation to control it for short term, we can have an `v1alpha2` version for spec change later. seems your changes below doesn't leverage this yet.